kml/types_v8mm.hpp

```diff
@@ -0,0 +1,314 @@
+#pragma once
+
+#include "kml/types.hpp"
+
+namespace kml
+{
+// BookmarkDataV8MM contains the same fields as BookmarkDataV8 but with no compilations
```

Comment:
```suggestion
// BookmarkDataV8MM contains the same fields as BookmarkDataV8 but without compilations.
```
nit: Have dots at the end of sentences.

Reply:
Fixed.